src/core/Form/LabelText/LabelText.tsx

```diff
@@ -25,13 +25,16 @@ interface InternalLabelTextProps extends HtmlDivProps {
   labelSpanProps?: HtmlSpanProps;
   /** Render the wrapping element as another element */
   asProp?: asPropType;
+  /** Optional text that is shown after labelText. Will be wrapped in parentheses. */
```

Comment:
Should this be described more like "An optional input indicator text that is..." or "text to mark a field optional --" so that the purpose is clearer from the description.

Reply:
I find the latter better understandable "text to mark a field optional --", could be also phrased like "Text to indicate an optional field"

Reply:
This is now changed to:
```
/** Text to mark a field optional. Will be wrapped in parentheses and shown after labelText. */
```